app/views/liquid_embeds/show.html.erb

```diff
@@ -2,11 +2,7 @@
 <% cache "liquid_tag_styles_#{ForemInstance.deployed_at}", expires_in: 8.hours do #TODO: Render specific ltag class instead of everything %>
   <style><%= Rails.application.assets["ltags/LiquidTags.css"].to_s.html_safe %></style>
 <% end %>
-<% begin %>
-  <% @liquid_node = Liquid::Template.parse("{% #{params[:embeddable]} #{params[:args]} %}").root.nodelist.first %>
-  <%= @liquid_node.render({}) %>
-<% rescue %>
-  <% raise ActionController::RoutingError, "Not Found" %>
-<% end %>
+
+<%= @liquid_node.render({}) %>
```

Comment:
`render` shouldn't raise `StandardError` here, because the error checking is in parse (which calls new)? 
 
 If we _only_ populate @liquid_node to render it, can we also render in the controller, and just pass the rendered content to the view?

Reply:
@djuber

> `render` shouldn't raise `StandardError` here, because the error checking is in parse (which calls new)?

I don't know which specific error class it raises, but this is what it was rescuing before when it was in the template. It's just being explicit here (and was added by a Rubocop autocorrect rule).

> If we _only_ populate @liquid_node to render it, can we also render in the controller, and just pass the rendered content to the view?

 I wondered the same thing, but I assume the rest of the template content is important.

Reply:
Oh wait, I misunderstood your second question. Yes, that would provide identical behavior and make this more of a true refactoring.